batch2/batch/worker.py

```diff
@@ -84,15 +87,15 @@ def __init__(self, container, state=None, name=None):
 
 
 class Container:
-    def __init__(self, pod, name, spec):
-        self.pod = pod
+    def __init__(self, job, name, spec):
+        self.job = job
         self.name = name
         self.spec = spec
 
         image = spec['image']
         tag = parse_image_tag(self.spec['image'])
         if not tag:
-            log.info(f'adding latest tag to image {self.spec["image"]} for container {self.pod.name}/{self.name}')
+            log.info(f'adding latest tag to image {self.spec["image"]} for {self}')
```

Comment:
For the container config below, I would like to add a name that is something equivalent to `batch-1-job-1-input`. I can do this later.

Reply:
You mean the docker container config?

Reply:
Yes. It helps when logging into a worker and doing `docker ps -a` to know what container corresponds to what job when looking in the logs.

Reply:
Ah, I couldn't find it in the container config; it's an option to container.create.  Done.